### PR TITLE
[frameworks] add snowpack-react

### PR DESCRIPTION
**Pull request by coetry**
This PR adds framework detection for the Snowpack React template

### CODE REVIEW

File: packages/frameworks/src/frameworks.ts [hunk 1]
```diff
@@ -1561,6 +1561,64 @@ export const frameworks = [
...
+      },
+    },
+    dependency: '@snowpack/plugin-react-refresh',
+    devCommand: 'snowpack dev',
+    buildCommand: 'snowpack build',
+    getOutputDirName: async () => 'build',
+    defaultRoutes: [
+      {
+        src: '/static/(.*)',
+        headers: { 'cache-control': 's-maxage=31536000, immutable' },
```
Comment (styfle) [inline packages/frameworks/src/frameworks.ts:1609]: Is this documented?

File: packages/frameworks/src/frameworks.ts [hunk 2]
```diff
@@ -1561,6 +1561,64 @@ export const frameworks = [
...
+        continue: true,
+      },
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '/(.*)',
+        headers: { 'cache-control': 's-maxage=0' },
```
Comment (styfle) [inline packages/frameworks/src/frameworks.ts:1617]: What is the purpose of this header? We already purge static files for each deployment so I don't think its necessary.

File: packages/frameworks/src/frameworks.ts [hunk 3]
```diff
@@ -1561,6 +1561,64 @@ export const frameworks = [
     buildCommand: null,
     getOutputDirName: async () => 'public',
   },
+  {
```
Comment (styfle) [inline packages/frameworks/src/frameworks.ts:1564]: The order of this file matters, so I would move Snowpack above "Other"

File: packages/frameworks/src/frameworks.ts [hunk 4]
```diff
@@ -1561,6 +1561,64 @@ export const frameworks = [
...
+    name: 'React Snowpack',
+    slug: 'react-snowpack',
+    demo: 'https://react-snowpack-delta.vercel.app',
+    logo:
+      'https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/react.svg',
+    tagline:
+      'Unlike traditional React application tooling, Snowpack saves you from getting bogged down with complex bundler setups and configuration files.',
+    description: 'A React app, bootstrapped with create-snowpack-app',
```
Comment (styfle) [inline packages/frameworks/src/frameworks.ts:1572]: It seems like `create-snowpack-app` doesn't work by itself but it requires a template. Which template did you use? Do we need other framework detection entries for each template, for example Vue?

**Comment by EndangeredMassa** [comment]
@coetry this PR is pretty old at this point. If you'd like to re-open it after addressing the feedback, feel free!

**Pull request closed without merging**